.github/labeler.yml

```diff
@@ -14,9 +14,10 @@ commercetools:
 boilerplate:
 - packages/boilerplate/**/*
 
-# Add 'docs' label to any change to packages/core/docs files EXCEPT for the changelog
+# Add 'docs' label to any change to packages/core/docs files
 docs:
-- any: ['packages/core/docs/**/*', '!packages/core/docs/changelog/**/*']
+- packages/core/docs/**/*
+- packages/core/docs/.vuepress/**/*
```

Comment:
These changes are correct?

Reply:
I made this change few weeks ago and it popped again when I merged `main`, which means that it got lost somewhere when we moved branches.